electrical/break_beam.py

```diff
@@ -0,0 +1,26 @@
+import RPi.GPIO as GPIO
+
+BEAM_PIN = 17
+beam_broken = False
+
+def break_beam_callback(channel):
+    if GPIO.input(BEAM_PIN):
+        beam_broken = False
+        # print(beam_broken)
+
+    else:
+        beam_broken = True
+        # print(beam_broken)
+
+    return beam_broken

```

Comment:
You can compress this if statement into a single line as seen in the example below.
```suggestion
   
    return not GPIO.input(BEAM_PIN)

```